src/Compilers/VisualBasic/Portable/Parser/ParseExpression.vb

```diff
@@ -1291,6 +1291,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             Dim closeParen As PunctuationSyntax = Nothing
             TryEatNewLineAndGetToken(SyntaxKind.CloseParenToken, closeParen, createIfMissing:=True)
 
+            If argumentBuilder.Count < 2 Then
+                argumentBuilder.AddSeparator(InternalSyntaxFactory.MissingToken(SyntaxKind.CommaToken))
+
+                Dim missingExpression = SyntaxFactory.SimpleArgument(nameColonEquals:=Nothing, expression:=SyntaxFactory.IdentifierName(InternalSyntaxFactory.MissingIdentifier()))
+                missingExpression = ReportSyntaxError(missingExpression, ERRID.ERR_TupleTooFewElements)
```

Comment:
>missingExpression = ReportSyntaxError(missingExpression, ERRID.ERR_TupleTooFewElements) [](start = 16, length = 87)

This looks inconsistent with C# implementation, which attaches the error to the MissingIdentifier.